litrpc/proxy.pb.go

```diff
@@ -220,20 +331,24 @@ func file_proxy_proto_rawDescGZIP() []byte {
 	return file_proxy_proto_rawDescData
 }
 
-var file_proxy_proto_msgTypes = make([]protoimpl.MessageInfo, 4)
+var file_proxy_proto_msgTypes = make([]protoimpl.MessageInfo, 6)
 var file_proxy_proto_goTypes = []interface{}{
-	(*StopDaemonRequest)(nil),  // 0: litrpc.StopDaemonRequest
```

Comment:
This is probably a stupid comment, but just wanted to make sure this isn't a potential issue:

There is no problem here that this overwrites the old `msgTypes` values here with new types right (ie. value `0` was the type `litrpc.StopDaemonRequest` but now becomes `litrpc.BakeSuperMacaroonRequest`)? Just thinking if we'd have external services or something that uses the protos and that way become incompatible, or if this could somehow become a backwards compatibility issue if one downgrades LiT or so.

This is likely not something that could become an issue, but thought I'd just leave this comment to make sure that's the case.

Reply:
good question!

So I think that either:
- these indices are only used within this file. So if you scroll up to where `file_proxy_proto_msgTypes` is used, you can see that it is only used by the message types once the types are known and are not used on the wire. 
- _or_, they are used on the wire but the mapping is conveyed via the `file_proxy_proto_rawDesc` descriptor above.

either way -  gRPC protos are backwards compatible & you can test this by running Lit with this PR but compiling litcli with master & then see that the `litcli getinfo` or `litcli stop` calls work just the same no matter if litcli is compiled with this branch or master branch :)